<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Install</class>
 <widget class="QWidget" name="Install">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:&quot;blue&quot;;</string>
     </property>
     <property name="text">
      <string>installation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="outputConsole">
     <property name="font">
      <font>
       <family>SpaceMono Nerd Font</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Shows current operation:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="install_btn">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:&quot;cyan&quot;;</string>
     </property>
     <property name="text">
      <string>Start installation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
